lib/bloom_api.rb

```diff
@@ -21,19 +21,22 @@ module BloomApi
   class ProviderNotFound < Exception; end
 
   # The base url for the bloom api.
-  BASE_URL = "www.bloomapi.com"
-  NPI_SEARCH_PATH = "/api/search/npi"
+  mattr_accessor :base_url, :npi_search_path, :api_key
```

Comment:
mattr_accessor is a rails extension to modules. Adding this prevents the gem from being used outside of a rails app. It might be better to manually create these accessors.

```
def base_url
  @@base_url
end

def base_url=(url)
  @@base_url = url
end
```